doc-content/enterprise-only/bpmn/bpmn-artifacts-ref.adoc

```diff
@@ -0,0 +1,22 @@
+[id='bpmn-artifacts_{context}']
+= BPMN2 artifacts in process designer 
+image:BPMN2/grn_star_sm.png[]
+Artifacts are used to provide additional information about a process. An artifact is any object depicted in the BPMN2 diagram that is not part of the process workflow. Artifacts have no incoming or outgoing flow objects.The purpose of artifacts is to provide additional information required to understand the diagram. The artifacts table lists the artifacts supported in the legacy process designer.
```

Comment:
Are you intending for that image/icon to be there? Seems odd. If by using it you mean that this section applies to the legacy designer, I'd nix the icon and just say so, or maybe it's apparently.